corehq/apps/translations/utils.py

```diff
@@ -20,15 +20,17 @@ def get_file_content_from_workbook(wb):
 
 def update_app_translations_from_trans_dict(app, trans_dict):
     if isinstance(app, LinkedApplication):
-        app_translation_dict = app.linked_app_translations
+        app_translation_dicts = [app.linked_app_translations, app.translations]
```

Comment:
@orangejenny you know why we do this? update two dicts for linked apps?

Reply:
Linked applications can use different translations than the master app. This was done for REACH

Reply:
Yeah, I think 45d13110ce522e3593fa44bec721bd67d98694f8 should be reverted so that linked apps *only* override `linked_app_translations`.

Reply:
@orangejenny i replicated the original implementation in that commit. So looks like that is expected?

Reply:
yep, my mistake